charlesinto/iReporter
# #162531609 Get a redflag record

#### What does this PR do?

- get a specific red-flags record

#### Description of Task to be completed?
- implement jwt authentication
- verify user token
- get a red-flags for an admin user
- get a red-flags for a specific user

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `GET` request to `http://localhost:7000/api/v1/red-flags/:id`
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint response should be structured like this- 



`

#### What are the relevant pivotal tracker stories?
[#162531609](https://www.pivotaltracker.com/story/show/162531609)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/26615085/49702224-2967df80-fbf6-11e8-9c35-289366976765.png)



#### Questions:
N/A
